source/_integrations/zwave_js.markdown

```diff
@@ -324,16 +325,16 @@ Some features can be accessed from the menu of integration itself. As they are n
 
 #### Network devices
 
-The following features can be accessed from the device panel of any Z-Wave device on your network aside from the controller:
+The following features can be accessed from the device panel of any Z-Wave device on your network aside from the adapter:
 
 ![Z-Wave device panel](/images/integrations/z-wave/z-wave-device-info.png)
 
 - **Configure:** Provides an easy way to look up and update configuration parameters for the device. While there is an existing action for setting configuration parameter values, this UI may sometimes be quicker to use for one-off changes.
 - **Re-interview:** Forces the device to go through the interview process again so that Z-Wave-JS can discover all of its capabilities. Can be helpful if you don't see all the expected entities for your device.
-- **Rebuild routes:** Forces the device to rediscover its optimal route back to the controller. Use this if you think you are experiencing unexpected delays or RF issues with your device. Your device may be less responsive during this process.
-- **Remove failed:** Forces the controller to remove the device from the controller. Can be used when a device has failed and it can't go through the normal exclusion process.
-- **[Statistics](https://zwave-js.github.io/node-zwave-js/#/api/node?id=quotstatistics-updatedquot):** Provides statistics about communication between this device and the controller, allowing you to troubleshoot RF issues with the device.
-- **Update:** Updates a device's firmware using a manually uploaded firmware file. Only some devices support this feature (controllers and devices with the Firmware Update Metadata Command Class).
+- **Rebuild routes:** Forces the device to rediscover its optimal route back to the adapter. Use this if you think you are experiencing unexpected delays or RF issues with your device. Your device may be less responsive during this process.
```

Comment:
```suggestion
- **Rebuild routes:** Discovers new routes between the adapter and the device. Use this if you think you are experiencing unexpected delays or RF issues with your device. Your device may be less responsive during this process.
```
The old text was not necessarily correct - I don't trust the Z-Wave firmware to actually find optimal routes.